app/models/view/reports/ReportSource.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.view.reports;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import models.view.impl.ChromeScreenshotReportSource;
+
+/**
+ * View model of {@link models.internal.reports.ReportSource}. Play view models are mutable.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ */
+@JsonTypeInfo(
+        use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.PROPERTY,
+        property = "type")
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = ChromeScreenshotReportSource.class, name = "ChromeScreenshot"),
```

Comment:
`CHROME_SCREENSHOT`?

Reply:
I had kept it Pascalcase (as with `HTML` and `PDF`) because typescript constants are styled like that. I suppose I could keep it capital so that the types are the same as on other endpoints, I just thought the translation code is more readable on the java side with these annotations (vs. handrolled on the typescript side).

Reply:
Not a problem as long as it's not contradicting any existing usage in MP and we keep it consistent.